src/product-card.css

```diff
@@ -0,0 +1,82 @@
+.product-card {
+  box-sizing: border-box;
+  justify-content: center;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__picture {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 36px;
+}
+
+.product-card__description {
+  margin: 0 16px;

```

Comment:
sep paddings for .product-card, no need to apply horizontal margins for each inner element